actionview/lib/action_view/rendering.rb

```diff
@@ -92,12 +92,16 @@ def rendered_format
       # Find and render a template based on the options given.
       # :api: private
       def _render_template(options) #:nodoc:
-        variant = options[:variant]
+        variant = options.delete(:variant)
```

Comment:
`:variant` is not used by `view_renderer` and is only needed by `lookup_context`. To make clear that it's not used further, it's removed from options.

The same with `:assigns`.